javacord-api/src/main/java/org/javacord/api/interaction/SlashCommandInteractionOptionsProvider.java

```diff
@@ -168,11 +168,11 @@
      *
      * @return An Optional with the double value of such an option if it exists; an empty Optional otherwise
      * @deprecated Use {@link #getOptions() getOptions().get(0).getNumberValue()}
-     *     or {@link #getOptionNumberValueByIndex(int) getOptionNumberValueByIndex(0)} instead.
+     *     or {@link #getOptionDecimalValueByIndex(int) getOptionDecimalValueByIndex(0)} instead.
      */
     @Deprecated
     default Optional<Double> getFirstOptionNumberValue() {
```

Comment:
Won't mark every place but the same should be done for `Second` and `Third` NumberValue
```suggestion
    default Optional<Double> getFirstOptionDecimalValue() {
```

Reply:
I intentionally left those out, as they are deprecated anyway.
Renaming them now would immediately remove the old ones and introduce new methods that are right-away deprecated.
That felt strange to me, so I kept them with the old name.
What do you think?

Reply:
Right, that would be a bit strange. But that applies for both options, having inconsistent naming again or like you said introducing new methods that are right away deprecated. But inconsistent naming might be a better than a breaking change for this case, so leave them as they are